README.md

```diff
@@ -1,20 +1,21 @@
-# Project Name
+# Apple clone webpage
 
-> One paragraph statement about the project.
+> This project is a clone of apple webpage.
```

Comment:
- [ ] Improve your Readme file. It should be descriptive of the project. You are using the correct template but now you need to fill it with information about your project. You can add details about what you did in this project, why you did it, how you did it...

- [ ] Delete the section that you are not using.